src/ocad-reader/index.js

```diff
@@ -131,4 +131,24 @@ class OcadFile {
       : { x: 0, y: 0, m: 1 }
     return new Crs(scalePar)
   }
+
+  getBounds(projection = v => v) {
```

Comment:
Great idea, I have missed this method in for example [ocad-tiler](https://github.com/perliedman/ocad-tiler/)!